uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestTest.php

```diff
@@ -364,4 +367,126 @@ private function getDefaultParams($params = []) {
     ];
     return array_merge($defaultParams, $params);
   }
+
+  public function testBalanceChangeIsUpdatedForAlreadyCreatedLeaveRequestWhenBalanceHasChangedAndChangeBalanceParameterIsTrueAndLeaveRequestDatesDidNotChange() {
```

Comment:
Wow, this is quite a long method name :D 
What about this slightly shorter version?

`testBalanceChangeIsUpdatedForAnExistingLeaveRequestWhenChangeBalanceParameterIsTrueAndDatesDidNotChange`